addons/web/static/src/views/fields/many2many_tags/many2many_tags_field.js

```diff
@@ -95,6 +98,20 @@ export class Many2ManyTagsField extends Component {
             },
         });
 
+        this.openMany2xRecord = useOpenMany2XRecord({
+            resModel: this.relation,
+            activeActions: {
+                create: this.props.canCreate && this.props.createDomain,
```

Comment:
Not sure this is useful, but it's wrong anyway. `create` must be a boolean, not a domain, which will always be considered truthy...